nengo_ocl/operators.py

```diff
@@ -35,7 +36,9 @@ def __init__(self, Y, Y_in, beta, gamma, tag=None):
 
     @classmethod
     def convert_to(cls, op):
-        if isinstance(op, DotInc):
+        if nengo_version <= (2, 3, 0) and isinstance(op, Copy):
```

Comment:
One thing we'll have to answer one day is how much legacy stuff we want to support. I can see this kind of stuff building up over time, and at some point we'll want to clean it out to uncomplicate things.

To facilitate that, should we have some way of marking where we have legacy things? Maybe just a comment with the word "legacy" in it. I don't think that word is used much otherwise.